mindsdb/integrations/handlers/replicate_handler/usage_text2video.md

```diff
@@ -0,0 +1,80 @@
+| animation_prompts | string  | 0: a beautiful portrait of a woman by Artgerm, trending on Artstation | Prompt for animation. Provide 'frame number : prompt at this frame', separate different prompts with '|'. Make sure the frame number does not exceed the max_frames. |
++-------------------+---------+-----------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+
+Now, you can use the established connection to query your ML Model as follows:
+```sql
+SELECT *
+FROM video_ai
+WHERE animation_prompts='a human and animals are friends by Asher Brown Durand, trending on Artstation'
+USING
+ max_frames=119;
+```
+
+### OUTPUT
+```sql
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| output                                                                                                                                                                                                                                                                                                                    | prompt                                 |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| https://replicate.delivery/pbxt/gSgRjNlxIgJWBB8KeebKeRmBjZx0wqX7JC41U0pvIfPCYVzEB/out.mp4 | a human and animals are friends by Asher Brown Durand, trending on Artstation |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------+
+```
+# TODO 
+Checkout generated video [here](./assets/animals.mp4) as above link will not work, reason is given below.
+
+- IMPORTANT NOTE: PREDICTED **URL** will only work for **24 hours** after prediction.
+
+> Note: Replicate provides only a few free predictions, so choose your predictions wisely. Don't let the machines have all the fun, save some for yourself! 😉

```

Comment:
The `usage_*.md` files contain a lot of duplicated text, especially the introduction to the model. Let's please merge all of these possible use cases under one single `.md`.

Reply:
is it necessary??

Because add more content in single `.md ` file is not likely for reader

Reply:
@ZoranPandovski @martyna-mindsdb what do you think is best in terms of readability for new contributors?

Reply:
@paxcema @parthiv11 
In this `replicate_handler` folder, I'd suggest linking all `usage_*.md` files in the `README.md` file.

When we create a doc page for the Replicate handler, we should include all these `usage_*.md` files in one `mdx` file (having chapters like LLM, Audio, etc.), as @paxcema mentioned.

Reply:
@martyna-mindsdb @paxcema i have added two files **`usage_all.md`** and `**allInOne.md**`.

Just check and let me know which can be used, or something else..

Reply:
@parthiv11 
Usually, README.md file is the one users look at first. Please add these links to README:
```
## Usage
To use this handler and connect to a Replicate cluster in MindsDB, you need an account on Replicate. Make sure to create an account by following this [link](https://replicate.com/signin?next=/account/api-tokens).

- [audio](./usage_audio.md)
    - [Audio Generation](./usage_audio.md#audio-generation)
    - [Custom Audio Cloning](./usage_audio.md#custom-audio-cloning)
- [img2text](./usage_img2text.md)
    - [Image Captioning](./usage_img2text.md#image-captioning)
    - [Image Text Matching](./usage_img2text.md#visual-question-answering)
- [LLM](./usage_LLM.md)
- [text2img](./usage_text2img.md)
- [text2vid](./usage_text2video.md)
```

Reply:
@martyna-mindsdb  made changes as suggested,